src/__tests__/linter.test.ts

```diff
@@ -745,43 +743,30 @@ responses:: !!foo
 
   test('should report when a resolver is not defined for a given $ref type', async () => {
     const s = new Spectral({ resolver: new Resolver() });
+    const document = JSON.stringify({
+      'file-refs': [{ $ref: './models/pet.yaml' }, { $ref: '../common/models/error.yaml' }],
+    });
 
-    const result = await s.run(invalidSchema, { ignoreUnknownFormat: true });
+    const result = await s.run(document);
 
     expect(result).toEqual([
       expect.objectContaining({
         code: 'invalid-ref',
         message: "No resolver defined for scheme 'file' in ref ./models/pet.yaml",
-        path: ['paths', '/pets', 'get', 'responses', '200', 'content', 'application/json', 'schema', '$ref'],
+        path: ['file-refs', '0', '$ref'],
         severity: DiagnosticSeverity.Error,
       }),
       expect.objectContaining({
         code: 'invalid-ref',
         message: "No resolver defined for scheme 'file' in ref ../common/models/error.yaml",
-        path: ['paths', '/pets', 'get', 'responses', 'default', 'content', 'application/json', 'schema', '$ref'],
+        path: ['file-refs', '1', '$ref'],
         severity: DiagnosticSeverity.Error,
       }),
     ]);
   });
 
-  test('should support YAML merge keys', async () => {
```

Comment:
not sure why we had this test :man_shrugging: 
It's about testing YAML merge keys, but we have a JSON document.